src/index.js

```diff
@@ -35,28 +35,10 @@ class MarkdownShortcuts {
         name: 'header',
         pattern: /^(#){1,6}\s/g,
         action: (text, selection) => {
-          switch (text.trim()) {
-            case '#':
-              this.quill.formatLine(selection.index, 0, 'header', 1)
-              break
-            case '##':
-              this.quill.formatLine(selection.index, 0, 'header', 2)
-              break
-            case '###':
-              this.quill.formatLine(selection.index, 0, 'header', 3)
-              break
-            case '####':
-              this.quill.formatLine(selection.index, 0, 'header', 4)
-              break
-            case '#####':
-              this.quill.formatLine(selection.index, 0, 'header', 5)
-              break
-            case '######':
-              this.quill.formatLine(selection.index, 0, 'header', 6)
-              break
-          }
+          const size = text.trim().length
```

Comment:
Because we get actual text of markdown pattern we can calculate header size from it's length.

Reply:
This is great, much cleaner.